### PR TITLE
[WIP] Coffea-casa topcoffea notebook

### PR DESCRIPTION
**Pull request by zorache**
Hi, this is a work-in-progress version of a coffea-casa compatible notebook running the topeft analysis. 

Outstanding:
- [ ] Currently don't have all the root files / configuration files do not match the root files 
- [ ] Need plotting to check results are working

### CODE REVIEW

**Comment by zorache** [comment]
@oshadura

**Comment by oshadura** [comment]
@GonzalezFJR could please advise Zora on how to resolve two bullet points she raised in PR description? Thank you so much in advance!!! (cc: @kenbloom)

File: cfg/samples.cfg [hunk 1]
```diff
@@ -1,4 +1,4 @@
-path : /home/xuan/coffea/trees/
+#path : /home/xuan/coffea/trees/
```
Comment (oshadura) [inline cfg/samples.cfg:1]: @GonzalezFJR what would you recommend to do here? Can we comment it out or better to remove it?

**Review by oshadura — COMMENTED** [review]
@GonzalezFJR I think Zora forgot to ask what to do with the path in the config file.

**Comment by oshadura** [comment]
Resolved in https://github.com/TopEFT/topcoffea/pull/33 (based on Zora work)

**Pull request closed without merging**